### PR TITLE
feat(inputdropdown): configure @emotion to enable using Component Sel…

### PR DESCRIPTION
**Pull request by masoudmanson**
Github issue: #386 

**InputDropdown**

- Configure @emotion/babel-plugin to enable using Component Selectors within Styled components

**DropdownMenu**

- Prevent Backspace from deselecting selected dropdown options.

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -17,18 +23,14 @@ module.exports = {
...
-          // (thuang): Temp fix for Github issue:
-          // https://github.com/storybookjs/storybook/issues/10231#issuecomment-728038867
-          ...config.resolve.alias,
-          "@emotion/core": toPath("node_modules/@emotion/react"),
-          "emotion-theming": toPath("node_modules/@emotion/react"),
-        },
-      },
+    // (thuang): Temp fix for Github issue:
+    // https://github.com/storybookjs/storybook/issues/10231#issuecomment-728038867
+    config.resolve.alias = {
```
Comment (seve) [inline .storybook/main.js:28]: I'd prefer creating a new object to return than modifying args, but not sure if y'all do it differently

**Review by seve — APPROVED** [review]
Not the requested reviewer, but this looks good to me!

File: .storybook/main.js (hunk 1 above)
Comment (masoudmanson) [reply .storybook/main.js:28]: Agreed, I'll just roll it back :D

File: src/core/DropdownMenu/index.tsx [hunk 2]
```diff
@@ -166,6 +166,12 @@ const DropdownMenu = <
                   ref={params.InputProps.ref}
                   search={search}
                   autoFocus
+                  // (masoudmanson): This prevents Backspace from deselecting selected dropdown options.
```
Comment (tihuan) [inline src/core/DropdownMenu/index.tsx:169]: Thanks for adding the comment!

File: src/core/InputDropdown/style.ts [hunk 3]
```diff
@@ -48,7 +48,7 @@ const inputDropdownStyles = (props: InputDropdownProps): SerializedStyles => {
         color: #000;
       }
 
-      .styled-label {
```
Comment (tihuan) [inline src/core/InputDropdown/style.ts:51]: Are we able to remove `styled-label` from classes prop now that we can use component selector 🙏 ?

Thank you!

**Review by tihuan — APPROVED** [review]
Woohoo! Thanks so much for adding component selector and fixing the issues, Masoud! What a relief lol LGTM  🚀

File: src/core/InputDropdown/style.ts (hunk 3 above)
Comment (masoudmanson) [reply src/core/InputDropdown/style.ts:51]: Aah yeah, for sure, I forgot about that. 😵‍💫 
I will update the PR

**Pull request merged**